content/integrations/azure.md

```diff
@@ -73,3 +121,4 @@ Please make sure you have enabled diagnostics on your VMs. Diagnostics allows VM
    [2]: https://portal.azure.com
    [3]: mailto:support@datadoghq.com
    [4]: https://management.azure.com
+   [5]: https://manage.windowsazure.com
```

Comment:
take a look at https://github.com/DataDog/documentation/blob/master/content/integrations/elasticsearch.md, scroll to the bottom. there is a block that says: <%= get_metrics_from_git()%>. This uses whatever git_integration_title is set to in the frontmatter at the top to grab the metrics csv from dogweb